ParcelInvoiceMaker/ParcelInvoiceMaker/Model/ParcelInfomation.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ParcelInfomation.swift
+//  ParcelInvoiceMaker
+//
+//  Created by Kyeongmo Yang on 3/6/24.
+//
+
+import Foundation
+
+class ParcelInformation {
+    let address: String
+    let receiverInfomation: ReceiverInfomation
+    let costInfomation: CostInfomation
+    
+    init(address: String, receiverInfomation: ReceiverInfomation, costInfomation: CostInfomation) {
+        self.address = address
+        self.receiverInfomation = receiverInfomation
+        self.costInfomation = costInfomation
+    }
+}
+
+struct ReceiverInfomation {
+    var receiverName: String
+    var receiverMobile: String
```

Comment:
struct 이름이 ReceiverInfomation 이기 때문에 안의 프로퍼티들은 receiver 에 대한 정보를 담고 있다고 생각할 수 있을 것 같아요.
그렇다면 중복된 이름은 지워도 되지 않을까요
```suggestion
    var name: String
    var mobileNumber: String
```

Reply:
생각안하고 옮겼다보니 그 부분을 놓쳤네요! 지적 감사합니다. 🫢